admin_scripts/ubuntu/ubuntu_16.04/chrome-specific/chrome_kiosk_disable_restore.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -e
+
+
+if [ $# -ne 2 ]
+then
+    echo "Dette script skal bruge to parametre: brugernavn og hjemmeside"
+    exit -1
+fi
+
+user=$1
+site=$2
+
+
+# Create exec script for Chrome
+cat <<CHROME-EXEC > /home/$user/.chrome.sh
+#!/usr/bin/env bash
+
+sed -i 's/"exited_cleanly":false/"exited_cleanly":true/' /home/$user/.config/google-chrome/Default/Preferences
+sed -i 's/"exit_type":"Crashed"/"exit_type":"None"/' /home/$user/.config/google-chrome/Default/Preferences
+sed -i 's/"restore_on_startup":[0-9]/"restore_on_startup":0/' /home/$user/.config/google-chrome/Default/Preferences
```

Comment:
This too should be a Chrome policy. (Is [RestoreOnStartup](https://www.chromium.org/administrators/policy-list-3#RestoreOnStartup) enough by itself, or will Chrome still try to recover from crashes?)